### ISSUE TITLE
Name is too long (maximum is 30 characters)

### ISSUE THREAD

**Issue by divyenduz** [prisma/prisma-cloud-feedback#237]
<img width="1234" alt="screen shot 2018-12-18 at 12 06 43 am" src="https://user-images.githubusercontent.com/746482/50107426-b6471480-0258-11e9-9ff1-5216cda9d753.png">

This error is very confusing as this happens with server name of 20 characters because internally Prisma cloud uses `<server-name>-<10-digit-hash>` to create the database name but that crosses the 30 character limit of Heroku. 

This error should say 20 characters to cause less confusion or DB name can be totally auto generated to be under 30 characters.

**Comment by chuckntaylor** [comment]
Just hit this same little snag. Although, this must be a fairly recent issue. I'm following a tutorial series where the instructor uses a name of 24 characters without issue. However, as @divyenduz points out, anything above 20 characters currently throws this error.

**Comment by divyenduz** [comment]
@chuckntaylor : Yes, I think this as introduced when we solved another bug regarding global uniqueness of workspaces around Heroku. It is a regression and relatively recent.
